mreg/api/v1/history.py

```diff
@@ -15,7 +15,8 @@ def default(self, o):
         if isinstance(o, Model):
             return model_to_dict(o)
 
-        return super().default(o)
+        # TODO: #485 We should never ever hit this. Should we fail?
+        return super().default(o)  # pragma: no cover
```

Comment:
Can't recall the reason for this.

Reply:
Pruned in https://github.com/unioslo/mreg/pull/484/commits/014e25b6030a05adf9622694539b92a38f71a2b1